src/components/task-plan/index.cjsx

```diff
@@ -82,7 +71,6 @@ PlanShell = React.createClass
       id={id}
       store={TaskPlanStore}
       actions={TaskPlanActions}
-      saved={@saved}
```

Comment:
Hm, I think the saved prop is actually a misnomer, as it only triggers when saving a new item, it should probably be "created", and I was actually going to remove it after this PR, since I don't think it's used anywhere else.